libs/doctrine-retry-bundle/tests/MysqlProxyTestTrait.php

```diff
@@ -58,24 +61,15 @@ function (bool $shouldFail) use ($toxiproxy, $mysqlProxy): void {
         ]);
 
         return DriverManager::getConnection([
-            'url' => $proxyMysqlHostPort,
+            'driver' => 'pdo_mysql',
```

Comment:
getConnection nově nemá url option, dal by se tam použít UrlParser, ale tím jak se to tady furt skládá a rozděluje, tak mě to přišlo jako zbytečně zamotaný

Reply:
DBAL ma primo DSN parser https://github.com/doctrine/dbal/blob/HEAD/UPGRADE.md#bc-break-removed-the-url-connection-parameter ktery jde jednoduse zavolat a result pouzit jako parametry pro `getConnection`

Reply:
Jo uz chapu, ten test trait sam potrebuje nejaky parametry z toho conneciton stringu. OK.

Reply:
Jo pardon, myslel jsem DsnParser, místo UrlParser